app/src/main/res/menu/downloads_completed.xml

```diff
@@ -22,5 +22,67 @@
         android:menuCategory="container"
         android:icon="@drawable/ic_refresh"
         app:showAsAction="always" />
+    <item
```

Comment:
Again here, this also feels quite redundant. Creating a menu xml file that contains sorting options, to be included on all screens, would be more elegant.